matrix-sdk-android-rx/src/main/java/im/vector/matrix/rx/RxSession.kt

```diff
@@ -56,10 +56,10 @@ class RxSession(private val session: Session) {
                 }
     }
 
-    fun liveBreadcrumbs(): Observable<List<RoomSummary>> {
-        return session.getBreadcrumbsLive().asObservable()
+    fun liveBreadcrumbs(onlyJoinedRooms: Boolean): Observable<List<RoomSummary>> {
```

Comment:
As we are returning RoomSummary, shouldn't we use the RoomSummaryQueryParams?

Reply:
Yrs, I've thought about it, will do the change